package.json

```diff
@@ -67,10 +67,7 @@
     "gzip-js": "~0.3.2",
     "gzip-size": "^3.0.0",
     "jasmine-check": "^1.0.0-rc.0",
-    "jest": "^21.3.0-beta.4",
-    "jest-config": "^21.3.0-beta.4",
-    "jest-jasmine2": "^21.3.0-beta.4",
-    "jest-runtime": "^21.3.0-beta.4",
```

Comment:
We weren't using these anymore

Reply:
`expect` is still in here in beta, is that used?

Reply:
Probably not..